german/Leichen/Leichen.md

```diff
@@ -6,6 +6,7 @@ werfen die beiden Leichen aus dem Boot.
 Sie schnaufen einmal kräftig durch während ihnen tausend Gedanken durch den Kopf gehen...
 
 Schnell, aber unbemerkt versuchen Sie an Land zu kommen, aber es scheint nicht näher zu kommen....
+Stattdessen sehe ich immer wieder einen kleinen Clownfisch-Piranha-Mischling an mir vorbeischwimmen und genüsslich an meinen Sachen nagen. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.